### DISCUSSION TITLE
Bass Cleff - Incorrect Notes???

### DISCUSSION THREAD

**Discussion by skuhn002** [0xfe/vexflow#1624] [Q&A]
I have been building a simple web application to practice sight-reading ([link](https://github.com/skuhn002/sight-read.git)), but I am getting incorrect notes on the bass cleff. Is anyone else having that issue?

**Comment by AaronDavidNewman** [comment]
This sounds like a bug that was fixed here:
https://github.com/0xfe/vexflow/commit/770015fae3ac0f3a24f8cfa15a0a728058db2d94

According to the notes, it looks like this is in the last 2 builds of vexflow 4.
